target_chains/ton/contracts/contracts/common/storage.fc

```diff
@@ -37,8 +35,6 @@ global int governance_data_source_index; ;; u32
         .end_cell();
 
     cell data_sources_cell = begin_cell()
-        .store_dict(data_sources)
-        .store_uint(num_data_sources, 32)
```

Comment:
we are removing these just because they are not needed?

Reply:
yup and they are not used anywhere